javascriptv3/example_code/lambda/actions/invoke.js

```diff
@@ -3,14 +3,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 import { InvokeCommand, LambdaClient, LogType } from "@aws-sdk/client-lambda";
+import { toUint8Array } from "@aws-sdk/util-utf8";
 import { createClientForDefaultRegion } from "../../libs/utils/util-aws-sdk.js";
 
 /** snippet-start:[javascript.v3.lambda.actions.Invoke] */
 const invoke = async (funcName, payload) => {
   const client = createClientForDefaultRegion(LambdaClient);
   const command = new InvokeCommand({
     FunctionName: funcName,
-    Payload: JSON.stringify(payload),
+    Payload: toUint8Array(payload),
```

Comment:
You'll still need JSON.stringify wrapped around the payload. This invoke function is meant to accept either a plain string or JSON object.

Reply:
I'm actually not convinced Uint8Array is the correct typing here. Trying to track down some more info.

Reply:
The [lambda model](https://github.com/aws/aws-sdk-js-v3/blob/main/codegen/sdk-codegen/aws-models/lambda.json#L6304) declares the type as a Blob. Blobs exist in the [browser](https://developer.mozilla.org/en-US/docs/Web/API/Blob), but aren't native to node.

For the browser, the type must match one of the types of [options.body in the Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Request/Request);

For Node, the type must match one of the types of [a chunk](https://nodejs.dev/en/api/v18/http/#requestwritechunk-encoding-callback).

Both of these support plain strings. This issue isn't actually an issue in that case, so I'm going to close it.